### ISSUE TITLE
"Failed to get game data from the server": ban?

### ISSUE THREAD

**Issue by manuelcanizares** [finndev/PokeBuddy#1222]
Hi everybody! And thanks for the awesome work you do everyday! From yesterday I cannot login in pokemon go and "Failed to get game data from the server" appears every time I try to enter. Does it mean I got banned? I have never received a soft ban so far. I don't abuse pokebuddy as I always wait the time I would need to come back home by car or by airplane.

**Comment by Demolition1986** [comment]
Well that message is that you're perma banned.. Nothing to do, just to create another account and fock it all with bot.

**Comment by manuelcanizares** [comment]
F**k! Is there any way to remove the perma ban?

**Comment by gusterwoei** [comment]
Yes, when you became the CEO of Niantic

**Comment by Demolition1986** [comment]
xoaxoaoxoaoxaooxaox No way to get out of perma ban.. Just sending an apeal with a cry words maybe they reisntated your account, better and fast is make another new account boting and getting lv 30 asap.

**Comment by manuelcanizares** [comment]
I don't want to wait one month 😁 so contacting niantic trying to have your profile back is completely useless? I have read something about how to have your profile back like trying to do a new account and then log in with your old one once you are inside the game, but they all seem some kind of bulls**t

**Comment by Demolition1986** [comment]
We need find Niantc corporation, then we go to the office and try to go inside, talks with the bots that are banning us.. Lets do it?

**Comment by manuelcanizares** [comment]
Just give me the time to fly from Italy to US since I don't have teleporting powers like the bot 😁😁😁

**Comment by barbaricacidman** [comment]
all my botted accounts are down but my real is working! :(

**Comment by Demolition1986** [comment]
@barbaricacidman Did u used spoof on the real one?

**Comment by ShupengHu** [comment]
My main account (lvl 40) was banned last week and my new account is lvl 22 with bot now. So easy. I will stop botting unitl lvl 30 for this account.

**Comment by barbaricacidman** [comment]
no never. strange as i havent botted them in probably a week or 2 due to recent issues but i guess its the way it is

only ones that are showing as "Failed to get game data from the server" are the ones i ran on pokebuddy

**Comment by manuelcanizares** [comment]
The problem is not how fast we can level up but how long our botted account can resist without being banned. If we arrive at level 25-30 and then we stopped using bot, I think Niantic will ban us sooner or later. There is no a safe way to use bot so far I suppose.

**Comment by barbaricacidman** [comment]
i had 4 that were all between level 20-25. been using the level 25 manually for a week now which hurts the most! lol

**Comment by Demolition1986** [comment]
Yeah, i just think that, is no way to bot and stay with account safe xaoxoax. I dont know how they are getting all bot ppl, maybe they have like 20k of indian guys checking bot ppl, idk xoaooa. I have 2 account lv 25 with bot and stop to bot until get 30. If i get both banned, bye pogo.
